.github/workflows/checks.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Update rubygems and install gems
         run: |
-          gem update --system 3.3.26 --silent --no-document
+          gem update --system --silent --no-document
```

Comment:
I'm not sure why this is breaking on JRuby 9.2... It might be worth bumping to 9.4 since that's what we expect to ship in puppetserver 8.

Reply:
Ahh, yes I think probably that old version of JRuby isn't supported by the new Bundler. I am testing out 9.4.0.0 jruby locally and will push up the change when those tests pass.

Reply:
If 9.4.0.0 goes sideways, then you might have better luck with the version facter is using https://github.com/puppetlabs/facter/blob/c0ea419a816f289f1574fb39c50fb0ed55d4e330/.github/workflows/unit_tests.yaml#L25